ee/modules/500-operator-trivy/images/operator/patches/README.md

```diff
@@ -27,6 +27,10 @@ This patch adds the ability to specify CA for scan jobs via environment variable
 
 ### 005-cis-benchmark-on-startup.patch
 
-The first check begins instantly when the operator starts
+The first check begins instantly when the operator starts.
 
 [Issue](https://github.com/deckhouse/deckhouse/issues/5174)
+
+### 006-new-metrics.patch
+
+This patch adds new prometheus metrics for reports.
```

Comment:
Please add more details - which metrics do we add.
`adds primaryLink metric for reports` at least something like this